config-hbase.sh

```diff
@@ -3,7 +3,7 @@
 # . /build/config.sh
 
 HBASE_DIST="http://apache.cs.utah.edu/hbase"
-#HBASE_DIST="http://archive.apache.org/dist/hbase"
+HBASE_DIST="http://archive.apache.org/dist/hbase"
```

Comment:
Seems the utah archive just keeps 1 dir max.  Switching back to the apache site is the only choice.